### ISSUE TITLE
Active Project?

### ISSUE THREAD

**Issue by powerbilayeredmap** [plasma-umass/browsix#63]
Is this still an active project? If i'm looking to run R code (and possibly some ggplot2 outputs) directly in the web browser, is this tool a possibility?

**Comment by emeryberger** [comment]
yes, it is very much an active project. we will be merging a major update soon (@bpowers). however, no plans to implement an X server, as far as I am aware.

**Comment by powerbilayeredmap** [comment]
Just curious, the comment on X server is related to ggplot2? ignoring ggplot2 for the moment, do you think running R code on your tool is realistic?

**Comment by emeryberger** [comment]
yes. in principle, R can be compiled via the emscripten toolchain to generate javascript or web assembly (which the next version of Browsix will support).
